### PR TITLE
Add note about super-linter

### PR DESCRIPTION
**Pull request by eitsupi**
Suggested by https://github.com/r-lib/lintr/issues/766#issuecomment-1074788133

I recently set up supler-linter in https://github.com/rocker-org/rocker-versioned2 and it seems to be working well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,6 +249,11 @@ If your project is on GitHub, you could take advantage of GitHub Actions and the
 usethis::use_github_action("lint-project")
 ```
 
+#### super-linter ####
+
+[Super-Linter](https://github.com/github/super-linter) and [MegaLinter](https://megalinter.github.io/latest/),
+which performs linting for various files, includes lintr for R.
```
Comment (MichaelChirico) [inline README.md:255]: "various files" --> "various languages"?

File: README.md [hunk 2]
```diff
@@ -249,6 +249,11 @@ If your project is on GitHub, you could take advantage of GitHub Actions and the
...
+They execute the `lintr::lint()` function on the `.r` or `.rmd` files included in the project.
```
Comment (MichaelChirico) [inline README.md:256]: case insensitive?

Comment (eitsupi) [reply README.md:256]: It does not seem to be case sensitive. Is there a better way to describe it?

Comment (eitsupi) [reply README.md:256]: "R and R Markdown files" ?

Comment (MichaelChirico) [reply README.md]: that sounds better yes

**Review by MichaelChirico — APPROVED** [review]
Thanks!

**Pull request merged**